core/kazoo_number_manager/src/knm_util.erl

```diff
@@ -126,3 +127,9 @@ read_fixture({'ok', Contents}, _F) ->
     kz_util:to_list(Contents);
 read_fixture({'error', 'enoent'}, F) ->
     throw({'error', 'missing_fixture', F}).
+
+%% TODO
+%% this should be replaced with a call to elibphonenumber
+%% when/if we integrate that lib or do it ourselves
+-spec prefix_for_country(ne_binary()) -> ne_binary().
+prefix_for_country(<<"US">>) -> <<"+1">>.
```

Comment:
no conversion involved here. we are not normalizing numbers.
even if/when we use libpn, regex will always be there for the cases where libpn doesn't work like dialplans